edenia/antelope-tools
# Feat/improve message and fix eos rate link

### Improve undiscoverable BPs empty state message and fix EOS Rate link

### What does this PR do?

- Resolve #1362
- Resolve #1366

### Steps to test

1. Run the project locally 
2. Go to Undiscoverable BPs page
3. Check the text
4. Go to a BP Profile with an EOS Rate rating
5. Test the link

### Screenshots

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/d2eef040-6294-4611-b9c9-c6d0b0335b8b)
